app/main.ipynb

```diff
@@ -100,18 +120,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "float_arr = np.arange(0, 11, dtype=np.float16)\n",
```

Comment:
The array `float_arr` is created with `np.arange(0, 11, dtype=np.float16)`, resulting in an array of length 11. Adjust the parameters to ensure the array length is between 5 and 10 as required.